lib/renderer/renderer-webpack.js

```diff
@@ -218,11 +218,12 @@ class Renderer {
         const clientFilename = "client.bundle.js";
         let memoryParsed;
         if (filePath.includes(".expressvue")) {
-            memoryParsed = path.parse(filePath.replace(".expressvue/", "").split(this.pagesPath)[1]);
+            memoryParsed = path.parse(filePath.replace(".expressvue" + path.sep, "").split(this.pagesPath)[1]);
         } else {
             memoryParsed = path.parse(filePath.split(this.pagesPath)[1]);
         }
-        const memoryBase = path.join(`/${this.baseUrl}/expressvue/bundles`, memoryParsed.dir);
+
+        const memoryBase = path.join(path.sep === '/' ? '/' : 'C:\\', `${this.baseUrl}/expressvue/bundles`, memoryParsed.dir);
```

Comment:
I was wondering if instead of doing a ternary on `path.sep` and hardcoding `C:\\` if `path.resolve()` could be used that will return the absolute path (containing the `C:\\`). The only caveat is that `path.resolve()` depends on the current dir it's run from. https://nodejs.org/api/all.html#path_path_resolve_paths

Reply:
Sure @625dennis you were the one that did this PR.. can you look at this?

Reply:
`path.resolve('/')` returns `C:\\` on windows. It's a nice solution.

Reply:
@625dennis could you update this on another PR so i can fix this issue?